src/explore-education-statistics-common/src/modules/full-table/services/tableBuilderService.ts

```diff
@@ -128,4 +136,10 @@ export default {
   getTableData(query: TableDataQuery): Promise<FullTable> {
     return dataApi.post('/tablebuilder', query);
   },
+  getTableDataForRelease(
+    query: TableDataQuery,
+    releaseId: string,
+  ): Promise<FullTable> {
+    return dataApi.post(`/tablebuilder?releaseId=${releaseId}`, query);
```

Comment:
Parameter is currently called `release`, not `releaseId`, but I can change it in the Admin API?

Reply:
I've renamed it to `releaseId` now but as discussed offline, I think we should consider removing it as it does nothing except stopping the Admin API from doing a check at the service layer that makes sure the Subject is for the latest release.